code/modules/power/singularity/singularity.dm

```diff
@@ -75,6 +75,19 @@
 	consume(user)
 	return 1
 
+/obj/singularity/attack_tk(mob/user)
+	if(!iscarbon(user))
+		return
+	var/mob/living/carbon/C = user
+	investigate_log("has consumed the brain of [key_name(C)] after being touched with telekinesis", "singulo")
+	C.visible_message("<span class='danger'>[C] suddenly slumps over.</span>", \
+	"<span class='userdanger'>As you mentally focus on the singularity, you feel your knowlage of the universe grow. So much raw information is there for the taking, and you won't waste a speck of it. Within fractions of a millisecond you absorb as much information as possible, more than even an AI would have access to, you feel like a god. Suddenly, the gravity of this situation dawns on you. As you feel your skull begining to collapse, you think to yourself. That was a really dense idea, wasn't it? </span>")
```

Comment:
```suggestion
		"<span class='userdanger'>As you focus on the singularity, you feel your knowledge of the universe grow. So much raw information is there for the taking; you won't waste a speck of it. Within fractions of a millisecond you absorb as much information as possible, more than even an AI would have access to, and you feel like a god. Suddenly, the gravity of this situation dawns on you. As you feel your skull beginning to collapse, you think to yourself. That was a really dense idea, wasn't it?</span>")
```
A few formatting + spelling things. Since this is a paragraph of flavor text, I would like to see some more people peer-revise it.

Reply:
Committed your suggested changes 👍